tests/udf.test/runit

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+cdb2sql --tabs --script ${1} ${CDB2_OPTIONS} - 2> actual 1>&2 <<'EOF'
```

Comment:
Are you missing `default` here?

Reply:
yeah it was bad copy pasta

Reply:
You could instead source tests/tools/compare_results.sh to avoid this boilerplate.